science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
+
+<figure class="figure-img">
+<img src="./ppa_test_result_lk_smooth_all-1_en.png" class="figure-img img-fluid" alt="Figure 4: Positivity rate after risk notification over time, smoothed and with confidence interval, averaged over all districts (thick) or nationwide (thin)."/>
+<figcaption aria-hidden="true">Figure 4: Positivity rate after risk notification over time, smoothed and with confidence interval, averaged over all districts (thick) or nationwide (thin).</figcaption>
+</figure>
+
+The positivity rate after a red warning is always greater than after a green warning (low risk with risk exposure). At high incidences, the positivity rate after lower risk without risk exposure is occasionally greater and at the same time more dependent on incidence (rises and falls faster) than after a red warning. A separate consideration of antigen rapid tests and PCR tests can shed light on this.
+
+To give an impression of the local differences, the following figure shows the courses for four randomly selected districts.
+
+
+<figure class="figure-img">
+<img src="./ppa_test_result_lk_smooth_sample-1_en.png" class="figure-img img-fluid" alt="Figure 5: Positivity rate after risk notification over time, smoothed and with confidence interval, for four randomly selected districts."/>
+<figcaption aria-hidden="true">Figure 5: Positivity rate after risk notification over time, smoothed and with confidence interval, for four randomly selected districts.</figcaption>
+</figure>
+
+Here, too, we clearly see the greater dependence of the positivity rate at lower risk without a risk exposure on the incidence: the green curve fluctuates more than the red or yellow curve.
+
+### 4.1 RAT
+
+The time course of the averaging of the positivity rates for rapid antigen tests (RAT) across all districts, broken down by risk notification, is included in the following figure.
+
+<figure class="figure-img">
+<img src="./ppa_test_result_lk_smooth_rat-1_en.png" class="figure-img img-fluid" alt="Figure 6: Positivity rate after risk notification over time, smoothed and with confidence interval, averaged over all districts (thick) or nationwide (thin) (RAT)."/>
+<figcaption aria-hidden="true">Figure 6: Positivity rate after risk notification over time, smoothed and with confidence interval, averaged over all districts (thick) or nationwide (thin) (RAT).</figcaption>
+</figure>
+
+This gives a similar picture as in the nationwide trend. A red warning in the CWA is clearly more likely to be followed by a time-related positive test result than for green warnings (low risk with risk exposure). In periods of high incidence, the positivity rates are also almost same and are not exceeded by the positivity rate at lower risk without a risk exposure. With the removal of the obligation to test, testing in recent months appears to be performed only in cases of reasonable suspicion, resulting in greater positivity rates even with relatively low (than in previous periods) incidences.
```

Comment:
In
"relatively low (than in previous periods) incidences"

"low" => "lower"

Reply:
Thanks for your suggestion, it has been implemented in https://github.com/corona-warn-app/cwa-website/pull/3363/commits/4c8e51aead1712b20d50c96a4cd72ac3b0e3bcb5